src/prefect/engine.py

```diff
@@ -192,14 +192,14 @@ async def create_then_begin_flow_run(
         raise RuntimeError(
             f"Cannot create flow run. Failed to reach API at {client.api_url}."
         ) from connect_error
-
     state = Pending()
     if flow.should_validate_parameters:
         try:
             parameters = flow.validate_parameters(parameters)
         except Exception as exc:
+            flow_run_logger(flow_run).exception(state.message)
```

Comment:
@peytonrunyan I think you meant 
```suggestion
            flow_run_logger(flow_run).exception("Validation of flow parameters failed.")
```
(note: no need to format the exception since `.exception` will log the full traceback)

Reply:
Also it looks like `flow_run` doesn't exist yet, so maybe my suggestion introduces too much complexity here

Reply:
Yup, but also I just realized that we don't have a flow run for a flow run logger until after line 212.